ui/pages/login/callback.js

```diff
@@ -28,8 +33,20 @@ export default function Callback() {
       } else {
         router.replace('/')
       }
-
       window.localStorage.removeItem('next')
+
+      let visitedOrgs = cookies.get('orgs') || []
```

Comment:
Just a heads up, we make cookies secure by default on the server at the moment when the request is using https. You'll have to add some logic to not do this for orgs cookies for this to work.

Reply:
resolved in chat